### ISSUE TITLE
Unable to build docker image

### ISSUE THREAD

**Issue by poffe** [didros/animus2influxdb#1]
Seems like the Dockerfile is missing or?

```
docker image build -t animus2influxdb:0.9 .
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /Users/christofferl/Code/Lab/animus2influxdb-master/Dockerfile: no such file or directory
```

**Comment by nonnos** [comment]
I’m having the same problem when trying to build the docker image...

**Comment by didros** [comment]
Sorry not seing that before. 
You have to "cd" in the folder where you clone the project to run the docker command provided.
E.g. 
`git clone https://github.com/didros/animus2influxdb.git`
`cd animus2influxdb`
`docker image build -t animus2influxdb:0.9 .`

**Comment by poffe** [comment]
No worries. This issue was solved by commit de89b53 - "Working in Docker" which added the missing Dockerfile. Thanks!
